managers/engineering_manager.md

```diff
@@ -27,28 +29,28 @@ Engineering Manager
 
 #### Meetings
 
-* Guide effectiveness of team meetings. Ensure agendas/purpose are clear & actions captured. 
+* Guide effectiveness of team meetings.
+* Ensure agendas/purpose are clear & actions captured.
 
 #### Determine team/team makeup & hiring needs
 
 * Ensure team has the people and skill sets needed to achieve their goals
-* Recruit for the team 
+* Recruit for the team
    * Source engineers
    * Motivate engineers to speak, do presentations, write blog posts, and contribute to open source projects (and do so oneself!)
-   * Participate in creating interview practice for Mariana Tek engineering
-   * Update interview loops & training engineers to use them for hiring 
+   * Update interview loops & training engineers to use them for hiring
    * Work with recruiting to ensure the loops run smoothly
    * Make the final hiring decision
```

Comment:
I think this is partially accurate - while I certainly think my input as a manager is important - I do think this is a bit of a broader decision making process (at least at MT). I think this is a holdover from the reference ladders more than anything

Reply:
I'm fine removing this